package com.dotori.v2.global.error.exception

import com.dotori.v2.global.error.ErrorCode

class InternalException : BasicException(ErrorCode.UNKNOWN_ERROR) {
}